tests/utils/print_vec_mat.hpp

```diff
@@ -207,5 +213,174 @@ void print_matrix( const grb::Matrix< T, B > & mat, size_t _limit = 0, const cha
 	std::cout << "==============" << std::endl << std::endl;
 }
 
+
+/**
+ * @brief Prints up to \p _limit rows and columns of matrix \p mat with optional heading \p head.
+ *
+ * @tparam T matrix data type
+ * @tparam B GraphBLAS backend storing the matrix
+ * @param mat matrix to print
+ * @param _limit max number of rows and columns to print (0 for all)
+ * @param head optional heading to print \b before the matrix
+ */
+template<
+	typename T,
+	enum grb::Backend B,
+	typename std::enable_if< std::is_void< T >::value, int >::type = 0
+>
+void print_matrix( const grb::Matrix< T, B > & mat, size_t _limit = 0, const char * head = nullptr ) {
```

Comment:
`& mat` -> `&mat`

would recommend passing in an output stream as argument, so that callers can decide which stream to output to